frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx

```diff
@@ -66,7 +66,7 @@ export const GlobalStyles = (): JSX.Element => {
         transparent
       );
 
-      ${themeSpecificSelectors}
+      ${getThemeSpecificCssVariables(theme)}
```

Comment:
I want to remove the CSS variables from GlobalStyles and reply on getMetabaseCssVariables, but that's quite error-prone so let's do that in the next PR. If we do that, there will be a lot less duplication between GlobalStyles and SDK.